### ISSUE TITLE
genomicVariations to return pathogenicity predictions

### ISSUE THREAD

**Issue by dglemos** [ga4gh-beacon/beacon-v2#76]
Hello,
This is more of a question. 
I need Beacon to return pathogenicity predictions such as [CADD](https://cadd.gs.washington.edu/) and [EVE](https://evemodel.org/. 
The current schema does not specify any field for these types of predictions. 
Could we have a new section under `VariantLevelData` just for in silico predictions?

**Comment by mbaudis** [comment]
@dglemos I'm +1 for this; logical addition (although we at some point have to comb over the whole variant model anyway, especial regarding its integration w/ VRS 1.3++).

```
  VariantLevelData:
    type: object
    properties:
      clinicalInterpretations:
        type: array
        items:
          $ref: '#/definitions/PhenoClinicEffect'
      phenotypicEffects:
        type: array
        items:
          $ref: '#/definitions/PhenoClinicEffect'
      pathogenicityPredictions:
        type: array
        items:
          $ref: '#/definitions/PhenoClinicEffect'
```

... like that? And would the `PhenoClinicEffect` class be sufficient? Needs then also respective definitions/examples.

@costero-e Could you have a look into organizing this into the branch setup, e.g. which feature branch this should go to and set one up if needed? And feed back to @dglemos to give her a place to craft a PR against. We can edit.

**Comment by costero-e** [comment]
Yes @mbaudis , I will take a look at it tomorrow if the day gives me a chance as I have been very busy lately

**Comment by costero-e** [comment]
Alright, I have crated a feature branch called **genomicVariations** and a subfeature branch called **genomicVariations-add-pathogenicity-predictions**. @dglemos you can modify this subfeature branch and PR to **genomicVariations** when you are ready

**Comment by dglemos** [comment]
@mbaudis @costero-e thank you for the response!
I'm working on the PR, it may take a few weeks to be ready though. 
I'm also planning on adding other changes such as molecular interactions and a small update to the frequency schema.

**Comment by dglemos** [comment]
Hi @costero-e,
I'm unable to push changes to the branch genomicVariations_add-pathogenicity-predictions. 
Can you please check the permissions?

**Comment by costero-e** [comment]
Hi @dglemos , I just sent you a collaboration request for you to be able to push into the branches. If you accept it you will be able to push now. Thank you.
